### ISSUE TITLE
Flexibilising the size of wellplates - Frontend adjustments

### ISSUE THREAD

**Issue by FabianMauz** [ComPlat/chemotion_ELN#1749]
After the basic work in the backend, the user should be able to define the size of wellplates themselves.

**_Specification & Implementation_**
- [x]  **Create new Wellplate**: At creation of a new wellplate the user should be able to set the height and width of the wellplates.  He should be supported either by a list of default sizes or by a saved size which was reckognized by his last entry.
- [x] **Download Import Template xlsx** : In the Attachments Tab there is the posiibility to download a xlsx which supports users to quickly import wellplates. Currently the xlsx is static and has a fixed size. We had to create a way to flexibilizise the xlsx
- [x] **Import template xlsx** : Based on a dynamic xlsx we have to check if the import algorithm must be changed
- [x] **Designer View** : The fixed 12x8 Grid in the Designer Tab must be flexibilisized
- [x] **List View** : The fixed 96 Table in List view must be flexibilisized
- [x] **Export Samples** : The functionality to export the samples of a wellplate must be checked if it gets a wellplate with other than 96 wells
- [x] **Print Wells** : The functionality to print the wells must be adapted. Up to a 96er well the printing works fine but for bigger wellplates the image is cropped. But is this a realistic usecase? After a discussion we decided to disable the print functionality for large wellplates.
- [x] **Report** : The reporting must be checked and adopted to flexible well sizes. Reporting of wellplates is currently not supported. Exporting of researchplans do currently not inculde wellplates.
- [x] **Researchplan** : Import Wellplate as table to "body" part must be enchanted
- [x] **Split Wellplates** : The split wellplate functionality always splits to 96er wellplates. That must be changed.
- [x] **Create Screen from Wellplate** :  Ask Pierre how to handle this. After a discussion we will look into that more detailed in another issue.
- [x] **Create Wellplate from Samples** : Always creates 96er wellplates, but can then be changed. But is more than 96 samples are clicked always 96er wellplates are created without the option to change the size.

**_BugFixing_**
- [ ] After making changes the yellow Save button  does not appear.
- [x] After creating a new wellplate there is no save shortcut in the top right area, only after changing the name.

_**Documentation**_
- [ ]  Update the user documentation
- [ ] Update the technical documentation
